src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -266,6 +266,33 @@ class CommandRequestImpl : public CommandImpl,
    */
   const CommandParametersPermissions& parameters_permissions() const;
 
+  /**
+   * @brief Adds interface to be awaited for by sdl request command
+     @param interface_id interface which SDL expects to response in given time
```

Comment:
@okozlovlux I cant see giving time in method signature. I suppose that this methods does not know anything about time. And they should not. Please remove such from description